src/picknik_ur_base_config/objectives/setup_mtc_open_lever_handle_door_config.yaml

```diff
@@ -0,0 +1,23 @@
+SetupMTCOpenLeverHandleDoor:
```

Comment:
Could we remove this file as well? No longer needed.

Reply:
Also removed.